bubble-wrap-style.yaml

```diff
@@ -2389,9 +2506,8 @@ layers:
                         }
                     }
                 draw:
-                    mapzen_icon_library:
+                    mapzen_icon_library: {}
```

Comment:
What's the deal with these lines (here and below) setting empty an empty object for the `draw` block? Can these just be removed or is that doing something? I'd expect it would just be a no-op when the layers are merged together.

Reply:
I set to `draw: mapzen_icon_library: {}`  because it's not obvious to me if I can have a sublayer of a parent layer without a draw block. So it was just me being caution & clever at the same time.